src/table/table.scss

```diff
@@ -367,6 +412,42 @@
     flex-basis: $iui-l * 2;
   }
 
+  &.iui-sticky {
+    position: sticky;
+    z-index: 1;
+  }
+
+  .iui-shadow-right,
```

Comment:
Maybe `iui-shadow-left` and `iui-shadow-right` should be in a separate mixin. There is no reason to nest this css inside cell.

Reply:
Moved